<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="etykieta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AHOJ! Jam jest pirat Roberts Straszliwy i mam dla Ciebie zagadkę!</string>
    </property>
   </widget>
   <widget class="QLabel" name="etykieta_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jest nią tajemna liczba od 1 do 99. Na odgadnięcie jej masz 6 prób.</string>
    </property>
   </widget>
   <widget class="QLabel" name="etykieta_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wprowadź liczbę i naciśnij przycisk Sprawdź!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblKomunikat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(komunikat)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSprawdz">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprawdź!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(status)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionZakoncz"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZakoncz">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
